app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
+    location = db.Column(db.String(64))
+    date = db.Column(db.DateTime)
+    agency = db.Column(db.String(64))
+    picture = db.Column(db.String(64))
+    description = db.Column(db.Text)
+
+    def __init__(self, vehicle_id, location, date, agency, picture, description):
```

Comment:
This init assumes that each of these fields is required. If this is the case, we should explicitly make them required using `nullable=False` (see https://github.com/hack4impact/gpcah/blob/0e41db2c4da1964bc84cb81afedca972ef87831a/app/models.py#L285)